src/org/labkey/test/util/StudyHelper.java

```diff
@@ -566,4 +582,40 @@ public boolean isSpecimenModuleActive()
         VIAL,
         SPECIMEN
     }
+
+    public enum Panel
+    {
+        studyGeneralSetup("General Setup", null),
+        studyWizardParticipantList("Participants", "Participant Group"),
+        studyWizardMouseList("Mice", "Mouse Group"),
+        studyWizardDatasetList("Datasets", "Label"),
+        studyWizardVisitList("Visits", "Label"),
```

Comment:
If you make this enum extend an interface, it could support custom participant types (e.g. Pandas).
```suggestion
    public interface Panel
    {
        String getPanelTitle();
        String getLabelColumn();
    }

    public enum Panels implements Panel
    {
        studyGeneralSetup("General Setup", null),
        studyWizardParticipantList("Participants", "Participant Group"),
        studyWizardMouseList("Mice", "Mouse Group"),
        studyWizardDatasetList("Datasets", "Label"),
        studyWizardVisitList("Visits", "Label"),
```

Then maybe add a convenience method for participants:
```
    public static Panel participantList(String singular, String plural)
    {
        return new Panel()
        {
            @Override
            public String getPanelTitle()
            {
                return plural;
            }

            @Override
            public String getLabelColumn()
            {
                return singular + " Group";
            }
        };
    }
```

Reply:
Done.